eng/Versions.props

```diff
@@ -5,7 +5,8 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-    <PreReleaseVersionLabel>rc.1</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel Condition="'$(SuppressFinalPackageVersion)' != 'true'">rc.1</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel Condition="'$(PreReleaseVersionLabel)' == ''">preview.4</PreReleaseVersionLabel>
```

Comment:
Can we instead do this in a `.targets` so we don't have to split properties out of the .csprojs?

Reply:
Unfortunately no, this needs to happen in a props file as it's values are read in the Common props from the SDK, so setting it later would be too late. I am adding a test like @radical suggested, so at least we'll know if this is broken in a project because they aren't using a d.b.props file.

Reply:
Have 2 properties here - named whatever.

And then in the Directory.Build.targets set the `PreReleaseVersionLabel`.

This is better ordering in MSBuild IMO

Reply:
This is how arcade consumes this property - in a .targets.

https://github.com/dotnet/arcade/blob/382c993e9ab742d2292c9330e1b952c5b57aaa2c/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L131-L137

Reply:
That is technically possible, but I would be worried about the potential impact of those changes, especially when making these in a release branch as we don't know what changing the ordering of those properties could do. What would you say about doing this less risky change here, and then in main (which will have more bake time and validation) do the change you suggest?

Reply:
>  this needs to happen in a props file as it's values are read in the Common props from the SDK, so setting it later would be too late

I don't think this is true. Doing a `dotnet build /bl` and looking at the preprocess, the only places I see that use this property are the .csproj setting it, and the arcade line above.

> but I would be worried about the potential impact of those changes

If you are really concerned about the risk, we can clutter the 8 projects here like we are doing. But I don't think this is ideal. If you want to do it the correct way in main, I'm fine with that.

Reply:
Yeah, at this point my top priority is to de-risk RC1, so unless you feel strongly about it, I'd opt for keeping this as-is and then getting this refactored in main